packages/preview/tidbit-uoc/0.1.0/template/main.typ

```diff
@@ -0,0 +1,49 @@
+#import "@preview/tidbit-uoc:0.1.0": template
+
+#show: template.with(
+    subject: "Herramientas HTML y CSS I",
+    title: "PEC1: Desarrollo de una web",
+    subtitle: "Iniciando la asignatura con un proyecto web",
+    date: datetime(year: 2025, month: 6, day: 30),
+    author: "Daniel Ramos Acosta",
+)
+
+= Introducción  
+
+#lorem(700)
+
+= Imágenes
+
+Un tamaño recomendado es un ancho del 75 % de la línea de texto con una altura
+proporcional a la primera. Todas las imágenes deben incluir una leyenda.
+
+#figure(image("assets/logo.svg"), caption: [
```

Comment:
The template can't access files outside of its subdirectory. You should make a copy of the logo in the template if you want to use it here too, or use another demonstration image.